### ISSUE TITLE
Performance

### ISSUE THREAD

**Issue by nengwp** [Speechless-10308/WSC#1]
Hello, very good paper, wonderful mathematical reasoning.
I noticed that in the paper comparison, the performance of some methods is far from that described in the paper. Is it your own reproduction score? For example, the Promix method.

![Image](https://github.com/user-attachments/assets/54cee999-4613-401b-8f4d-680588d90156)

![Image](https://github.com/user-attachments/assets/65298a9f-eb2d-447c-90e2-b22b449a96f6)

**Comment by Speechless-10308** [comment]
Hello, for NLL, our result for other methods are mainly from [Twin Contrastive Learning with Noisy Labels](https://arxiv.org/pdf/2303.06930), [ULAREF: A Unified Label Refinement Framework for Learning with Inaccurate Supervision](https://openreview.net/pdf?id=ssFMq35UUY) and [Imprecise Label Learning: A Unified Framework for Learning with Various Imprecise Label Configurations](https://arxiv.org/pdf/2305.12715), which have the similar setting like ours (eg. Training epoch and backbone.) Specifically, the result for ProMix is from [ULAREF](https://openreview.net/pdf?id=ssFMq35UUY). The training epochs in the original Promix paper are 600, which is significantly longer than our 250 epochs. Due to time constraints, we did not reproduce the results of Promix in our settings, and used the results of ULAREF as a reference.

**Comment by nengwp** [comment]
Thank you for your quick reply, I understand.
In addition, I found that the values ​​of hyperparameters vary greatly for different datasets. Is this method sensitive to hyperparameters for different datasets?

**Comment by Speechless-10308** [comment]
The setting of the hyperparameters $\alpha$ and $\beta$ is related to the number of categories in the dataset. Denote $C$ is the number of class, setting the ratio of $\beta$ to $\alpha$ at $2C$ can often achieve a promising performance in general.
